### PR TITLE
LPS-138945 Apply Workflow Instance Tracker to instance items on Metrics

### PR DESCRIPTION
**Pull request by MarinhoFeliphe**
Story: [LPS-138937](https://issues.liferay.com/browse/LPS-138937)

### CODE REVIEW

**Comment by MarinhoFeliphe** [comment]
ci:test:relevant

**Comment by naraandrade** [comment]
ci:test:relevant

**Review by naraandrade — APPROVED** [review]

File: modules/dxp/apps/portal-workflow/portal-workflow-metrics-web/src/main/java/com/liferay/portal/workflow/metrics/web/internal/configuration/FFWorkflowMetricsConfigurationUtil.java [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Modified;
+
+/**
+ * @author Nara Andrade
+ */
+@Component(
+	configurationPid = "com.liferay.portal.workflow.metrics.web.internal.configuration.FFWorkflowMetricsConfiguration",
+	immediate = true, service = FFWorkflowMetricsConfigurationUtil.class
+)
```
Comment (rafaprax) [inline modules/dxp/apps/portal-workflow/portal-workflow-metrics-web/src/main/java/com/liferay/portal/workflow/metrics/web/internal/configuration/FFWorkflowMetricsConfigurationUtil.java:31]: Hey @naraandrade, please use just `service = {}` instead as we won't use the DI to get an instance of this service. Thx

//cc @MarinhoFeliphe

**Review by rafaprax — CHANGES_REQUESTED** [review]

**Comment by naraandrade** [comment]
ci:stop

**Comment by MarinhoFeliphe** [comment]
ci:test:sf

**Comment by MarinhoFeliphe** [comment]
ci:test:relevant

**Comment by naraandrade** [comment]
ci:test:relevant

**Comment by naraandrade** [comment]
ci:test:relevant

**Review by rafaprax — APPROVED** [review]

**Comment by jotascript** [comment]
Just started reviewing :)

**Review by jotascript — APPROVED** [review]

**Comment by naraandrade** [comment]
ci:forward

**Comment by anabuchmann** [comment]
Just started reviewing :)

**Comment by anabuchmann** [comment]
Manually forwarded here: https://github.com/brianchandotcom/liferay-portal/pull/107143

**Pull request closed without merging**